app/templates/people/person/personal-information/index.hbs

```diff
@@ -0,0 +1,93 @@
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-flow au-o-flow--large">
+    <div class="au-o-flow au-o-flow--tiny">
+      <AuToolbar>
+        <AuToolbarGroup>
+          <AuHeading>
+            Persoonlijke gegevens
+          </AuHeading>
+        </AuToolbarGroup>
+        <AuToolbarGroup>
+          <AuLink
+            @route="people.person.personal-information.edit"
+            @skin="button-secondary"
+            @icon="pencil"
+            @iconAlignment="left"
+          >
+            Bewerk
+          </AuLink>
+        </AuToolbarGroup>
+      </AuToolbar>
+      <AuHeading @level="2" @skin="3">
+        {{@model.person.givenName}} {{@model.person.familyName}}
+      </AuHeading>
+    </div>
+
+    <DataCard>
+      <:title>Persoonlijke gegevens</:title>
+      <:card as |Grid|>
+        <Grid as |Item|>
+          <Item>
+            <:label>Voornaam</:label>
+            <:content>{{@model.person.givenName}}</:content>
+          </Item>
+        </Grid>
+        <Grid as |Item|>
+          <Item>
+            <:label>Achternaam</:label>
+            <:content>{{@model.person.familyName}}</:content>
+          </Item>
+        </Grid>
+        <Grid as |Item|>
+          {{#if @model.person.firstNameUsed}}
+          <Item>
+            <:label>Roepnaam</:label>
+            <:content>{{@model.person.firstNameUsed}}</:content>
+          </Item>
+          {{/if}}
+        </Grid>
+
+      </:card>
+    </DataCard>
+
+    <DataCard>
+      <:title>Contactgegevens:
+        {{@model.person.mandatories.firstObject.mandate.governingBody.isTimeSpecializationOf.administrativeUnit.classification.label}}
+      </:title>
+      <:card as |Grid|>
+        <Grid as |Item|>
+          <Item>
+            <:label>Organisatie</:label>
+            <:content>
+              <AuLink
+                @model={{@model.person.mandatories.firstObject.mandate.governingBody.isTimeSpecializationOf.administrativeUnit.id}}
+                @route="administrative-units.administrative-unit">
+                {{@model.person.mandatories.firstObject.mandate.governingBody.isTimeSpecializationOf.administrativeUnit.name}}
+              </AuLink>
+            </:content>
+          </Item>
+        </Grid>
+        <Grid as |Item|>
+          <Item>
+            <:label>Email</:label>
+            <:content>{{@model.person.mandatories.firstObject.contacts.firstObject.email}}</:content>
+          </Item>
+          <Item>
+            <:label>Telefoonnummer</:label>
+            <:content>{{@model.person.mandatories.firstObject.contacts.firstObject.telephone}}</:content>
+          </Item>
+        </Grid>
+        <Grid as |Item|>
+          <Item>
+            <:label>Adres</:label>
```

Comment:
I think the address can be added to the same `<Grid>` as the email and phone number, right?

Reply:
We should use a different grid if we make contact fields optional throught the if. Otherwise it will change position when one attribute is missing.

Reply:
That doesn't really matter right? The only problem is that if all items in a grid are empty / hidden it looks weird, so that probably requires a large check as well. But I guess that won't happen in practice.

Reply:
Maybe we can check on @model.person.mandatories.firstObject?